packages/generator/templates/app/src/app/auth/components/LoginForm.tsx

```diff
@@ -0,0 +1,53 @@
+import {AuthenticationError, PromiseReturnType} from "blitz"
+import Link from "next/link"
+import {LabeledTextField} from "src/app/components/LabeledTextField"
+import {Form, FORM_ERROR} from "src/app/components/Form"
+import login from "../mutations/login"
+import {Login} from "../validations"
+import {useMutation} from "@blitzjs/rpc"
+
+type LoginFormProps = {
+  onSuccess?: (user: PromiseReturnType<typeof login>) => void
+}
+
+export const LoginForm = (props: LoginFormProps) => {
+  const [loginMutation] = useMutation(login)
+  return (
+    <div>
```

Comment:
```suggestion
    <>
```

The div is not needed here AFAIK